lib/ansible/utils/module_docs_fragments/netapp.py

```diff
@@ -48,9 +48,20 @@ class ModuleDocFragment(object):
       aliases: ['pass']
   https:
       description:
-      - Enable and disabled https
+      - Enable and disable https
       type: bool
       default: false
+  validate_certs:
+      description:
+      - If set to C(False), the SSL certificates will not be validated.
+      - This should only set to C(False) used on personally controlled sites using self-signed certificates.
+      default: true
+      type: bool
+  http_port:
```

Comment:
Should this just  be `port` since it's for http(s)?

Reply:
Some of the existing Netapp Modules (and some of the future ones) currently use 'port' for modifying the network port on the Ontap System. For instance, the net_port modules use to set the network port https://docs.ansible.com/ansible/latest/modules/na_ontap_net_port_module.html#na-ontap-net-port-module

Reply:
Thanks for the info